test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateTools.cs

```diff
@@ -19,6 +19,11 @@ public class GivenThatIWantToMigrateTools : PackageDependenciesTestBase
         [InlineData("Microsoft.VisualStudio.Web.CodeGenerators.Mvc", "1.0.0-preview3-final", "Microsoft.VisualStudio.Web.CodeGeneration.Design", ConstantPackageVersions.AspNetToolsVersion)]
         [InlineData("Microsoft.VisualStudio.Web.CodeGenerators.Mvc", "1.1.0-preview4-final", "Microsoft.VisualStudio.Web.CodeGeneration.Design", ConstantPackageVersions.AspNet110ToolsVersion)]
         [InlineData("Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design", "1.1.0-preview4-final", "Microsoft.AspNetCore.Mvc.Razor.ViewCompilation", ConstantPackageVersions.AspNet110ToolsVersion)]
+        [InlineData("Microsoft.VisualStudio.Web.BrowserLink.Loader", "1.0.0-preview2-final", "Microsoft.VisualStudio.Web.BrowserLink", "1.0.1")]
```

Comment:
Tests also should be fixed with 14.* style versioning for Microsoft.VisualStudio.Web.BrowserLink.Loader. 
-- see http://www.nuget.org/packages/Microsoft.VisualStudio.Web.BrowserLink.Loader